SocialCareCaseViewerApi/V1/Infrastructure/Worker.cs

```diff
@@ -15,26 +17,36 @@ public class Worker
         [Column("email")]
         [MaxLength(62)]
         [Required]
-        public string Email { get; set; }
+        public string Email { get; set; } = null!;
 
         [Column("first_name")]
         [MaxLength(100)]
         [Required]
-        public string FirstName { get; set; }
+        public string FirstName { get; set; } = null!;
 
         [Column("last_name")]
         [MaxLength(100)]
         [Required]
-        public string LastName { get; set; }
+        public string LastName { get; set; } = null!;
 
         [Column("role")]
         [MaxLength(200)]
-        public string Role { get; set; }
+        public string? Role { get; set; }
 
+        [Column("context_flag")]
+        [MaxLength(1)]
+        public string? ContextFlag { get; set; }
+
+        [Column("created_by")]
+        [MaxLength(62)]
+        public string? CreatedBy { get; set; }
+
+        [Column("date_start")]
+        public DateTime? DateStart { get; set; }
 
```

Comment:
EndDate that is nullable (so only set this when deactivating the worker), otherwise it's always null, for audit tracking